docs/source/guides/libraries.rst

```diff
@@ -1,91 +1,314 @@
++----------+----------+-----------------------+---------------------------------------------+-------------------------------------+
+| 2        | Yes      | **Android API Level** | API level supported by your NDK.            | ``27`` for android-ndk-r16b         |
+|          |          |                       | See the link under the table for details.   |                                     |
++----------+----------+-----------------------+---------------------------------------------+-------------------------------------+
+| 3        | Yes      | **Android NDK Path**  | Full path to unpacked NDK. Please           | ``/home/user/lib/android-ndk-r16b`` |
+|          |          |                       | ensure that path does not contain spaces.   |                                     |
++----------+----------+-----------------------+---------------------------------------------+-------------------------------------+
+| 4        | Yes      | **Java Package Name** | Package name that will be used for Java     | ``jp.co.soramitsu.iroha.android``   |
+|          |          |                       | interfaces generation. Note that the binary |                                     |
+|          |          |                       | also depends on chosen package name.        |                                     |
++----------+----------+-----------------------+---------------------------------------------+-------------------------------------+
+| 5        | No       | **Build Type**        | Defines build mode of binary part           | ``Debug`` or ``Release``            |
+|          |          |                       | of the library. ``Release`` is the default  |                                     |
+|          |          |                       | option.                                     |                                     |
++----------+----------+-----------------------+---------------------------------------------+-------------------------------------+
+
+`Android API levels <https://developer.android.com/guide/topics/manifest/uses-sdk-element.html#ApiLevels>`__
+
+Please use the same root part of Java package name for library build as you use for your Android project.
+For example, your project is located in a package called ``com.mycompany.androidapp``, so please consider to build the library in a
+package, which name starts with ``com.mycompany`` (e.g. ``com.mycompany.iroha``).
+
+A couple of launch commands examples:
+
+.. parsed-literal::
+    # build Java bindings and binary library for arm64-v8a in Release mode
+    ./android-build.sh arm64-v8a 27 /home/user/lib/android-ndk-r16b com.mycompany.iroha
+
+    # build Java bindings and binary library for x86 in Debug mode
+    ./android-build.sh x86 27 /home/user/lib/android-ndk-r16b com.mycompany.iroha Debug
+
+Build artefacts will be collected in ``lib`` directory near the script ``android-build.sh``.
```

Comment:
artifacts?

Reply:
Both are acceptable. Artefact in British English and artifact in US English.

Reply:
Please confirm if you insist on the change.